packages/flutter/test/painting/text_painter_rtl_test.dart

```diff
@@ -284,25 +286,31 @@ void main() {
       painter.getOffsetForCaret(const TextPosition(offset: 1, affinity: TextAffinity.upstream), Rect.zero),
       const Offset(10.0, 0.0),
     );
+
+    Offset offset = (Platform.isWindows) ? new Offset(7.0, 12.0) : new Offset(10.0, 10.0);
     expect( // between A and Alef, before the Alef
       painter.getOffsetForCaret(const TextPosition(offset: 1, affinity: TextAffinity.downstream), Rect.zero),
-      const Offset(10.0, 10.0),
+      offset,
     );
 
+    offset = (Platform.isWindows) ? new Offset(0.0, 12.0) : new Offset(0.0, 10.0);
     expect( // after the Alef
       painter.getOffsetForCaret(const TextPosition(offset: 2, affinity: TextAffinity.upstream), Rect.zero),
-      const Offset(0.0, 10.0),
+      offset,
     );
     expect( // after the Alef
       painter.getOffsetForCaret(const TextPosition(offset: 2, affinity: TextAffinity.downstream), Rect.zero),
-      const Offset(0.0, 10.0),
+      offset,
     );
 
+    TextBox textBox = (Platform.isWindows) ?
```

Comment:
final for `textBox` - and does TextBox have a const constructor that could be used?

Reply:
Using const for `TextBox` doesn't seem to work since `Platform.isWindows` isn't const.